src/Components/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -2,163 +2,108 @@ import {
   Box,
   BoxProps,
   Flex,
-  Input,
   InputProps,
+  LabeledInput,
   Select,
   SelectProps,
+  Spinner,
   Text,
 } from "@artsy/palette"
-import { useState } from "react"
-import { useSystemContext } from "System"
-import { getPhoneNumberInformation } from "./getPhoneNumberInformation"
 import { countries } from "Utils/countries"
-import { useDebouncedValue } from "Utils/Hooks/useDebounce"
-
-export interface PhoneNumber {
-  isValid: boolean
-  national?: string
-  regionCode?: string
-}
-
-export interface PhoneNumberValidationResult {
-  international: string
-  isValid: boolean
-  national: string
-  originalNumber: string
-  region: string
-}
+import {
+  PhoneNumberValidationResult,
+  usePhoneNumberValidation,
+} from "Components/PhoneNumberInput/usePhoneNumberValidation"
+import { ChangeEvent, useState } from "react"
+import { useDebouncedCallback } from "use-debounce"
 
 export interface PhoneNumberInputProps extends BoxProps {
-  phoneNumber: PhoneNumber
-  onPhoneNumberValidation: (arg: PhoneNumberValidationResult) => void
-  inputProps?: Partial<InputProps>
-  optional?: boolean
-  selectProps?: Partial<SelectProps>
-  error?: string
+  required?: boolean
+  error?: string | boolean
+  inputProps: InputProps
+  onValidation: (result: PhoneNumberValidationResult) => void
+  selectProps: Omit<SelectProps, "options">
 }
 
 export const PhoneNumberInput: React.FC<PhoneNumberInputProps> = ({
-  phoneNumber: { isValid, national, regionCode },
-  onPhoneNumberValidation,
+  required,
+  onValidation,
+  error,
   inputProps,
-  optional,
   selectProps,
-  error,
-  ...props
+  ...rest
 }) => {
-  const { relayEnvironment } = useSystemContext()
-
-  const [number, setNumber] = useState(isValid && national ? national : "")
-  const [countryCode, setCountryCode] = useState(
-    isValid && regionCode ? regionCode : "us"
-  )
-  const [focus, setFocus] = useState(false)
-  const [hover, setHover] = useState(false)
```

Comment:
I don't think this was particularly helpful. These are separate interactive elements and we should treat them as such.